authutils/auth_service_test.go

```diff
@@ -22,3 +22,16 @@ func TestCompareWithLatestBlockNumber(t *testing.T) {
 	assert.Equal(t, nil, err)
 
 }
+
+func TestCheckAllowedBlockDifferenceForToken(t *testing.T) {
+	config.Vip().Set(config.BlockChainNetworkSelected, "ropsten")
+	config.Validate()
+	currentBlockNum, _ := CurrentBlock()
+	err := CheckAllowedBlockDifferenceForToken(currentBlockNum.Add(currentBlockNum, big.NewInt(2000000)))
```

Comment:
As far as I understand the logic `CheckAllowedBlockDifferenceForToken` gets a date when token was issued in blocks. Then this date should be less then current date. But in test the date which bigger then current date is passed.
```suggestion
	err := CheckAllowedBlockDifferenceForToken(currentBlockNum.Sub(currentBlockNum, big.NewInt(2000000)))
```

Reply:
now its Just Cmp